Packs/AzureResourceGraph/Integrations/AzureResourceGraph/README.md

```diff
@@ -0,0 +1,182 @@
+Azure Resource Graph is an Azure service designed to extend Azure Resource Management by providing efficient and performant resource exploration with the ability to query at scale across a given set of resources.
+
+## Authorize Cortex XSOAR for Azure Resource Graph (Self-Deployed Configuration)
+
+To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal. To add the registration, see the [Microsoft article](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
+
+## Authentication
+For more details about the authentication used in this integration, see [Microsoft Integrations - Authentication](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication).
+
+- After authorizing the XSOAR App or the Self-Deployed Application, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields. After giving consent, the application has to have a role assigned so it can access the relevant resources per subscription.
```

Comment:
XSOAR App does not exist for this integration, please leave only the reference to Self-Deployed

Reply:
Updated in dff3feb